dev/docker-compose.yaml

```
@@ -0,0 +1,52 @@
+version: "3.8"
+services:
+  # Connect to prompt with `docker exec -it postgres_container sh -c "psql -U postgres -d tdp"`
```

Without `sh` it does not work ?
```suggestion
  # Connect to prompt with `docker exec -it postgres_container psql -U postgres -d tdp`
```

---

yes it does work, has been changed